cpu/stm32l1/periph/dac.c

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
the dac periph implementation has moved to a common place. Can you revert this change ?